lib/ansible/modules/cloud/amazon/route53_health_check.py

```diff
@@ -128,9 +141,18 @@
     state: absent
     fqdn: host1.example.com
 
+- route53_health_check:
+    state: present
+    type: CALCULATED
+    child_health_checks:
+      - "{{ my_health_check.health_check.id }}"
+      - "{{ another_health_check.health_check.id }}"
+    health_threshold: 2
+  register: my_health_check
 '''
 
 import uuid
+import ipaddress
```

Comment:
I'm unsure why for this. The feature you've added doesn't use it, it only appears you've replaced bits of the existing code with it. Is there a bug you're fixing?

Reply:
The CI report `ipaddress` is not part of python 2.X. I don't know what is the recommended way to fix it, should we add a dependency to the ipaddress package (https://pypi.python.org/pypi/ipaddress/1.0.18) or should we try to fix it without this module?